packages/xpetra/src/CrsMatrix/Xpetra_EpetraCrsMatrix.hpp

```diff
@@ -1015,9 +1006,6 @@ class EpetraCrsMatrixT <int, EpetraNode>
             // End of TMP
 
             out << "Node number of entries = " << getNodeNumEntries() << std::endl;
-            if (isFillComplete()) {
-              out << "Node number of diagonals = " << getNodeNumDiags() << std::endl;
-            }
             out << "Node max number of entries = " << getNodeMaxNumRowEntries() << std::endl;
```

Comment:
This reminds me that the ability to call getNodeMaxNumRowEntries or getNodeNumEntries before first fillComplete of a CrsGraph is one of the few things that makes it not thread safe to add entries to a graph.

Reply:
That's not relevant to this PR but it's good to keep in mind :D .